features/step_definitions/autocomplete_steps.rb

```diff
@@ -41,147 +46,171 @@
     when "Characters"
       "ellen"
     when "Relationships"
-      "stiel"
-    when "Your Tags"
+      "cast"
+    when "Your tags"
```

Comment:
`case` are indented in 2 different ways in the project: as deep as case, or 2 spaces from the start of the statement. Which style should I use?

We can get Hound to agree: http://rubocop.readthedocs.io/en/stable/cops_style/#stylecaseindentation